keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -232,20 +232,44 @@ _This section must be completed when targeting alpha to a release._
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes, but CNIs relying on the new field wont recognize it anymore
+  
+  Yes. One caveat here is that NetworkPolicies created with EndPort field set 
+  when the feature was enabled will continue to have that field set when the 
+  feature is disabled unless user removes it from the object. 
+  
+  If the value is dropped with the FeatureGate disabled, the field can only 
+  be re-inserted if feature gate is enabled again.
+
+  Rolling back the Kubernetes API Server that does not have this field
```

Comment:
Maybe clarify: The net impact of a rollback is that a port-range reverts to a single port.  This may break users, but that is inevitable in such a case.  It satisfies the fail-closed requirement.